shark/examples/shark_inference/stable_diffusion/resources/beta_model_db.json

```diff
@@ -0,0 +1,177 @@
+[
```

Comment:
Is there any need for de-duplicating this?

Reply:
My thinking was, when we're ready to replace the old files, it will be easier to simply overwrite the previous file with the beta file, instead of replacing individual entries.